### ISSUE TITLE
training loss (finetuning) curve 

### ISSUE THREAD

**Issue by MlWoo** [yangdongchao/RSTnet#6]
Very grateful for your work, as mentioned in readme, finetuning on fisher dataset is released, could you share the training loss curve?
